editors/sc-ide/widgets/help_browser.hpp

```diff
@@ -154,6 +157,9 @@ private slots:
     QSize mSizeHint;
 
     QAction *mActions[ActionCount];
+
+    bool mServerRunning = false;
+    int mServerPort = 12344;
```

Comment:
you could have two members here, but you really only need one if you use a default value of `-1` or some other invalid-by-definition port number; opinions differ on it, but I find this use of bools kind of wasteful since the semantics are obvious. and in fact, it makes the logic above (in setUpServer) more difficult to follow.